### ISSUE TITLE
Default code description in response

### ISSUE THREAD

**Issue by nonnib** [marshmallow-code/flask-smorest#112]
If I run the application from the quickstart tutorial with the following additons:
```py
app.config["OPENAPI_URL_PREFIX"] = "/doc"
app.config["OPENAPI_REDOC_PATH"] = "/redoc"
```
I can view the generated openapi documentation. However, the status says **200 undefined**.

Upon further examination it appears that the problem lies in the fact that no description is supplied  with the status code.

If I add the following change:

```py
@blp.route("/")
class Pets(MethodView):
    @blp.arguments(PetQueryArgsSchema, location="query")
    @blp.response(PetSchema(many=True), description=HTTPStatus(200).phrase)
    def get(self, args):
        return []
```
The status shows up correctly as **200 OK**.

I would like to suggest putting a default status description if one is not supplied.

In response.py#58:
```py
        if description is not None:
            resp_doc['description'] = description
        else:
            resp_doc['description'] = HTTPStatus(code).phrase
```

**Comment by lafrech** [comment]
Thanks. This makes total sense.

`description` is [required](https://swagger.io/specification/#responseObject) so that would be a bugfix.

Would you like to tackle this?

**Comment by nonnib** [comment]
Sure, I'll give it a go. Thanks!

**Comment by lafrech** [comment]
Closed in #113.
